### ISSUE TITLE
How to write an advice that executes whenever object fn throws exception

### ISSUE THREAD

**Issue by rvnath** [manokovacs/aopromise#1]
I am using db calls which return a Promise.  But if DB ops fail, they throw an exception, which gets handled by the .catch(fn(){...}) block.  Is there a way to execute an advice whenever there is an exception thrown?

I have also created a stackoverflow issue [here](http://stackoverflow.com/questions/40605590/nodejs-aop-perform-a-common-action-for-db-exceptions).

**Comment by manokovacs** [comment]
@rvnath, good point. Actually its not yet documented, but you can do it. I gave a more detailed answer on stackoverflow, where [I link this test ](https://github.com/manokovacs/aopromise/blob/master/test/aopromise.test.js#L227) which checks this actual feature. Let me know if that works for you!
